src/Raven.Server/Documents/Handlers/Admin/AdminDocumentsMigrationHandler.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.Threading.Tasks;
+using Raven.Server.Documents.Sharding;
+using Raven.Server.Json;
+using Raven.Server.Routing;
+using Raven.Server.Utils;
+using Sparrow.Json;
+
+namespace Raven.Server.Documents.Handlers.Admin
+{
+    public class AdminDocumentsMigrationHandler : DatabaseRequestHandler
```

Comment:
let's rename this to `ShardDatabaseHandler`

Reply:
AdminShardingHandler

Reply:
done